### PR TITLE
fix(DB/Dungeon): Dire Maul West

### PR DESCRIPTION
**Pull request by Knindzagxg**
Changes:

Min, Max & Modifier Damage for whole Dungeon.
Added flags for each boss, so they will appear with skull instead regular level elite.

All checks have been done trough Classic Realms and Youtube videos.

How to test:
Apply SQL
Restart Server and delete your Cache.
Make sure to add very very basic gear towards Dungeon ilevel.

Before you start checking damage:
.mod hp 50000
.learn 1908 (Uber heal over time)

If there is any issue with damage and if you get instantly killed, lemme know.

### CODE REVIEW

**Comment by Geekena** [comment]
damage and spells are ok
bosses as well
been lost there and double tested all but its all good

**Comment by Knindzagxg** [comment]
Whole dungeon is confusing, but once you do it few times its very easy to find way xP

File: data/sql/updates/pending_db_world/rev_1598878239120359200.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+/* RARE */
+UPDATE `creature_template` SET `rank` = 2, `mindmg` = 496, `maxdmg` = 692, `DamageModifier` = 1.02 WHERE `entry` = 11467;
+
+/* BOSS */
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 689, `maxdmg` = 771, `DamageModifier` = 1.01 WHERE `entry` = 11489;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 684, `maxdmg` = 741, `DamageModifier` = 1.01 WHERE `entry` = 11487;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 692, `maxdmg` = 757, `DamageModifier` = 1.01 WHERE `entry` = 11488;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 675, `maxdmg` = 714, `DamageModifier` = 1.01 WHERE `entry` = 11496;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 695, `maxdmg` = 723, `DamageModifier` = 1.01 WHERE `entry` = 11486;
+UPDATE `creature_template` SET `type_flags` = 4, `mindmg` = 684, `maxdmg` = 776, `DamageModifier` = 1.01 WHERE `entry` = 14506;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1598878239120359200.sql:39]: https://www.azerothcore.org/wiki/Dealing-with-SQL-files#sql-basics

**Review by Helias — CHANGES_REQUESTED** [review]

File: data/sql/updates/pending_db_world/rev_1598878239120359200.sql (hunk 1 above)
Comment (Knindzagxg) [reply data/sql/updates/pending_db_world/rev_1598878239120359200.sql]: I forgot about it xP
Sorted now, will edit other as well xP

**Review by Helias — APPROVED** [review]

**Pull request merged**